From e6a3ea5b2b5c520d1e69a8473cb4ac1361305e9d Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Tue, 20 Nov 2012 11:47:25 +0800
Subject: [PATCH 2293/2508] ENGR00234236 1588: fix kernel build warning

Warning: no return statement in function returning non-void.
fec_ptp_ioctl return zero when 1588 is not enable.

Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 drivers/net/fec_1588.h |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/drivers/net/fec_1588.h b/drivers/net/fec_1588.h
index bd3565a..9daf9de 100644
--- a/drivers/net/fec_1588.h
+++ b/drivers/net/fec_1588.h
@@ -217,7 +217,10 @@ static inline void fec_ptp_store_rxstamp(struct fec_ptp_private *priv,
 					 struct sk_buff *skb,
 					 struct bufdesc *bdp) {}
 static inline int fec_ptp_ioctl(struct fec_ptp_private *priv,
-				struct ifreq *ifr, int cmd) {}
+				struct ifreq *ifr, int cmd)
+{
+	return 0;
+}
 
 #endif /* 1588 */
 
-- 
1.7.7.4

